FirebaseStorage/Tests/SwiftIntegration/StorageIntegration.swift

```diff
@@ -705,6 +716,16 @@ class StorageIntegration: XCTestCase {
     waitForExpectations()
   }
 
+  private func signInAndWait() {
+    let expectation = self.expectation(description: #function)
```

Comment:
I wonder if `self` can be omitted here?

Reply:
nope. Variable used within its own initial value